examples/mini-app/src/index.css

```diff
@@ -175,7 +190,6 @@
     --text-on-surface-variant: var(--dark-on-surface-variant);
     --text-on-surface: oklch(0.93 0.0058 264.53);
     --border-stroke: var(--dark-stroke);
-    --play-btn-font: var(--white);
 
     --game-win: oklch(0.78 0.1418 166.22);
 
```

Comment:
The roulette color variables are defined in the light theme (lines 153-163) but are missing from the dark theme section. Consider adding dark theme variants here:

```css
/* Dark theme roulette colors */
--roulette-red: oklch(0.5273 0.1693 23.38);
--roulette-red-hover: oklch(0.5573 0.1693 23.38);
--roulette-black: oklch(0.243 0.0139 235.28);
--roulette-black-hover: oklch(0.273 0.0139 235.28);
--roulette-green: oklch(0.684 0.1218 166.22);
--roulette-green-hover: oklch(0.704 0.1218 166.22);
--roulette-bundle: oklch(0.1398 0.0062 214.42);
--roulette-bundle-hover: oklch(0.1698 0.0062 214.42);
--roulette-disabled: oklch(0.2399 0.0061 239.84);
--roulette-disabled-text: oklch(0.4138 0.014 271.18);
```